specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -0,0 +1,1939 @@
+              "headers": {
+                "Location": {
+                  "type": "string",
+                  "description": "Location and ID of the composed model. The status of composed model is specified in the status property at the model location."
+                }
+              }
+            },
+            "default": {
+              "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+        },
+        "x-ms-examples": {
+            "Compose custom models": {
+              "$ref": "./examples/ComposeModels.json"
+            }
+        }
+      }
+    },
+    "/prebuilt/businessCard/analyze": {
+      "post": {
+        "summary": "Analyze Business Card",
+        "description": "Extract field text and semantic values from a given business card document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png' or 'image/tiff'. Alternatively, use 'application/json' type to specify the location (Uri) of the document to be analyzed.",
+        "operationId": "AnalyzeBusinessCardAsync",
+        "consumes": [
+          "application/pdf",
+          "application/json",
+          "image/jpeg",
+          "image/png",
+          "image/tiff"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "includeTextDetails",
+            "in": "query",
+            "description": "Include text lines and element references in the result.",
+            "required": false,
+            "default": false,
+            "type": "boolean",
+            "x-nullable": false
+          },
+          {
+            "name": "locale",
+            "in": "query",
+            "description": "Locale of the business card. Supported locales include: en-AU, en-CA, en-GB, en-IN, en-US.",
+            "required": false,
+            "type": "string"
+          },          
+          {
+            "$ref": "#/parameters/FileStream"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Request is queued successfully.",
+            "headers": {
+              "Operation-Location": {
+                "type": "string",
+                "description": "URL containing the resultId used to track the progress and obtain the result of the analyze operation."
+              }
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Analyze receipt": {
+            "$ref": "./examples/BusinessCardBatch.json"
+          }
+        }
```

Comment:
This looks like an LRO. Can we add the `"x-ms-long-running-operation": true` flag for code gen?